### PR TITLE
fix: panel display font color and backlight

### LINKED ISSUE #4241: Transponder and Rudder display brightness
**Is your feature request related to a problem? Please describe.**

In ambient light, it is difficult to read the displays in both the Transponder and Rudder Trim displays.

**Describe the feature you'd like to see implemented**

I would expect the intensity to vary with the setting of the INTEG LT potentiometer on the pedestal. Is this possible?

**References**

AirbusA320Pro "The pedestal backlight illumination is controlled as @gkkes suggests. That's how the real aircraft works IRL".

**Additional context**

Evidence of lower illumination are evident here between the radio brightness and the TX and Rudder Trim intensity.
txlevel

Discord username (if different from GitHub): gkkes#1982

### PR DESCRIPTION
**Pull request by marcman86**
Fixes #4241

## Summary of Changes

-Backlight to RMP displays (copied from ATC)
-Brighter fontcolor to ATC display (copied from RMP)
-Brighter fontcolor and matching backlight to RTPI (copied from FCU) 

## Screenshots (if necessary)

Before:

After:

## References

image0

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1.) Spawn
2.) Power must be on
3.) Look at panels at different lighting conditions

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Benjozork** [comment]
Please fix the title.

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by St54Kevin** [comment]
"-Backlight to RMP displays (copied from ATC)"
Does this mean the display brightness are finally able to be controlled via the INTEG LT knob?

**Comment by marcman86** [comment]
> 
> 
> "-Backlight to RMP displays (copied from ATC)"
> Does this mean the display brightness are finally able to be controlled via the INTEG LT knob?

I just copied the colors, no logic. So no.

**Comment by Snapmatics** [comment]
> > "-Backlight to RMP displays (copied from ATC)"
> > Does this mean the display brightness are finally able to be controlled via the INTEG LT knob?
> 
> I just copied the colors, not the logic. So no.

Will try to do that with my PR (the logic).

**Comment by marcman86** [comment]
Increased the backlight brightness of the RTPI a bit.

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
 ## 0.7.0
+1. [DISPLAYS] RMP backlight, ATC and RTPI font color and backlight - @github (Marcman86)
 1. [MCDU] Fixed input and display issues on PERF/W&B and INIT pages - @felixharnstrom (Felix Härnström)
```
Comment (tyler58546) [inline .github/CHANGELOG.md:9]:
```suggestion
1. [MCDU] Fixed input and display issues on PERF/W&B and INIT pages - @felixharnstrom (Felix Härnström)
1. [DISPLAYS] RMP backlight, ATC and RTPI font color and backlight - @marcman86 (Marcman86)
```

**Comment by St54Kevin** [comment]
Category: QA Tester
Name: St54Kevin#3583
Date of testing: 08/04/2021
Version of the sim: 1.14.6.0
PR Tested: #4337 
Tier of testing: 1
Changes to observe: Pedestral displays

**Steps you've made to test the Pull Request:**
1. Loaded up at the Gate
2. Checked pedestral displays in different lighting situations

**Media:**
Daylight Before
image
Daylight After
image

**Issues (If Any):** - 

**Conclusions of the testing:** LGTM. final brightness settings can be adjusted with the bightness knob function

**Comment by DavidEGTE** [comment]
Category: QA Tester
Name: David#2177
Date of testing: 08/04/2021
Version of the sim: 1.14.6.0
PR Tested: #4337 
Tier of testing: 1

Changes to observe:
panel display font colour and backlight

Testing technique: Try-out

Steps you've made to test the Pull Request:
1. Spawned C&D
2. Played with random brightness settings

Flight notes (When done a full flight):
N/A
Media: Insert screenshots, recordings, or anything relevant to the review here.
N/A
Issues (If Any): Write any issues
N/A
Steps to achieve the issue:
N/A

Overall Rating: Good
Conclusions of the testing: LGMT

**Comment by marcman86** [comment]
>     1. When selected "OFF" the backlighting is still being displayed. (For future reference)

Fixed that :-)

**Comment by BritishAvgeek** [comment]
Category: QA Tester
Name: BritishAvgeek#1436
Date of testing: 09/04/2021
Version of the sim: 1.14.6.0
PR Tested: #4337
Tier of testing: 1

Changes to observe: Panel display font colour and backlight improvements

Testing technique: Spawn Cold and Dark on stand and check

Steps you've made to test the Pull Request:
1.Spawned C&D
2. In daylight, changed brightness settings and checked against previous A32NX to compare
3. Night time - checked again different brightness settings
4. Checked RMP Stack fix as mentioned in a test above being still slightly backlit - can confirm that was now off for me so appears fixed.
5. 
Flight notes (When done a full flight): N/A
Media: N/A

Issues (If Any): None
Steps to achieve the issue: N/A

Overall Rating: Good
Conclusions of the testing: no issues found, fix above has fixed the rmp stack backlight remaining on as mentioned on a previous test and everything worked as expected and described, thanks!

**Review by tyler58546 — APPROVED** [review]
LGTM

**Comment by PillowPilot22** [comment]
Category: QA Tester/Trial
Name: PillowPilot#1914
Date of testing: Current date 11/04/2021
Version of the sim: Version the tester uses
PR Tested: #4241 
Tier of testing: 1
Changes to observe: Panel display font colour and backlight improvements


Testing technique: spawn cold and dark on stand and check
Steps you've made to test the Pull Request:
1. spawned cold and dark
2. turned on the aircraft and checked in daylight
3. changed tiime to night and looked again
4. Checked the RMP stacks and they have been fixed
5.

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here.

Issues (If Any): This might just be me but i found that when loaded in i didnt have the A32NX textures
Microsoft Flight Simulator 11_04_2021 13_49_23

Steps to achieve the issue: Downloading this and extracting it into the A32NX

Overall Rating: Good/To be improved
Conclusions of the testing: Make a small descriptive to areas where it needs to be improved and recommendations.
Apart from that the issues have been fixed and RMP stack looking much better

**Comment by TheSquirtingPantin** [comment]
Category: QA Trial
Name: Discord User Rémi#8871
Date of testing: Current date 12/04/2021
Version of the sim: V 1.14.6.0
PR Tested: #4337
Tier of testing: 1

Changes to observe:
Backlighting of RMPs and RTPI. 

Testing technique:
1. RWY 08 spawning at KDEN
2. Checked the backlighting of both RMPs and RTPI

Flight notes (When done a full flight):N/A

Media: N/A

Issues (If Any): N/A

Overall Rating: Good
Conclusions of the testing: LGTM, good job !

**Pull request merged**